internal/services/cdn/cdn_frontdoor_profile_resource.go

```diff
@@ -122,7 +122,7 @@ func resourceCdnFrontDoorProfileRead(d *pluginsdk.ResourceData, meta interface{}
 	ctx, cancel := timeouts.ForRead(meta.(*clients.Client).StopContext, d)
 	defer cancel()
 
-	id, err := parse.FrontDoorProfileID(d.Id())
+	id, err := parse.FrontDoorProfileIDInsensitively(d.Id())
```

Comment:
this is coming from the Resource ID (which is being validated at import time) and so should be being parsed case-sensitively:

```suggestion
	id, err := parse.FrontDoorProfileID(d.Id())
```

Reply:
Fixed.